### ISSUE TITLE
[Feature Request]  Swipe Side of Page to Change Brightness

### ISSUE THREAD

**Issue by coltaku** [inorichi/tachiyomi#1720]
It would be awesome if you could add the ability to change the brightness inside the app with swiping up and down on the left (or right) side of the page, on the edge of the screen. This would be super convenient and would honestly work perfectly with this app on my opinion. This feature would make this app nearly perfect.

**Comment by dragneelfps** [comment]
This looks interesting.

**Comment by that-one** [comment]
But please, make it optional, maybe a toggle in the brightness& colour window or in the reader settings.

**Comment by coltaku** [comment]
Yeah, a toggle switch in the brightness/color filter tab would be perfect. I'm glad other people would also enjoy this feature being added.
